### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by brittcyr**
TODO in a followup

link to autogen docs
link to benchmarking
command for client testing
logos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,76 @@
...
+| CU | | |
+|Instruction size | | | |
+|Silent failures | Yes| Yes| No|
+|Token 22 | No| No| Yes|
+
+- Cranks were originally used in serum to address the need for solana programs to identify all accounts before landing on chain. This has become obsolete now that orderbooks pack all data into a predictable account
+- No trading fees in Manifest
+- Lot sizes restrict expressable prices. This meaningfully matters to orderflow through routers that have non-standard sizes. Manifest reduces the min trade size to atomic and increases the the range of expressable prices to cover all that are needed.
+- Anchor is great for starting on Solana, but more advanced programs should not take the compute tradeoff for the convenience
+- Rent is the big savings for Manifest. This enables the use case of smaller value tokens with less volume to have orderbooks.
```
Comment (DonDuala) [inline README.md:30]: mention realloc?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,76 @@
...
+| Predefined lot sizes |Yes |Yes |No|
+| Anchor |Yes |Yes |No|
+| Rent|2 SOL |2 SOL |.002 SOL|
+| License|GPL |Business |GPL|
+| OpenOrders separation for read locking| Yes | No | Yes |
+| Swap accounts| 10 | 8 | 7 |
+| CU | | |
+|Instruction size | | | |
+|Silent failures | Yes| Yes| No|
+|Token 22 | No| No| Yes|
```
Comment (DonDuala) [inline README.md:24]: Add wrapper architecture No, No, Yes?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,76 @@
...
+<pre>
+--------------------------------------------------------------------------------------------------------
+|                   Header                    |                               Dynamic                   |
+--------------------------------------------------------------------------------------------------------
+| BaseMint, QuoteMint, BidsRootIndex, ...     | Bid | Ask | FreeListNode | Seat | Seat | Bid | Bid | Ask|
+--------------------------------------------------------------------------------------------------------
+</pre>
+
+### Core vs Wrapper
+Manifest views the orderbook as an infrastructure layer primitive. Other orderbooks get bogged down by special feature requests from trading teams that ultimately make the program bloated and confusing. Manifest strives to only include features that are absolutely necessary to be in the base layer. Anything that can be handled at layers above on the stack will not be done in manifest. This simplification makes formal verification of the program feasible.
```
Comment (DonDuala) [inline README.md:56]: Manifest implements the orderbook as an infrastructure layer primitive and creates the purest form of risk exchange possible.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,76 @@
-# manifest
-The feeless order book exchange that supercharges your on-chain trading abilities
+# Manifest
+
+The feeless order book exchange that supercharges your on-chain trading
+abilities Manifest is the next generation of orderbooks on Solana. It is
+optimized for freedom. No more permissioned markets. No more trading fees. No
+more large 2 SOL rent to start a market. Capital efficiency built in.
```
Comment (DonDuala) [inline README.md:6]: No more expensive rent or gas costs to start a market

**Review by DonDuala — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brittcyr) [reply README.md:30]: realloc on its own isnt a big deal. effects mentioned elsewhere though

File: README.md (hunk 2 above)
Comment (brittcyr) [reply README.md:24]: You could write a wrapper for any, just not built for it

File: README.md (hunk 4 above)
Comment (brittcyr) [reply README.md:6]: gas is gonna be higher for manifest to reserve capacity early on

**Pull request merged**